core/cloudflow-operator-actions/src/main/scala/cloudflow/operator/action/Action.scala

```diff
@@ -427,6 +449,7 @@ final class ProvidedAction[T <: ObjectResource, R <: ObjectResource](
 final class ProvidedByLabelAction[T <: ObjectResource, R <: ObjectResource](
     val labelKey: String,
     val labelValues: Vector[String],
+    val app: CloudflowApplication.CR,
```

Comment:
I do see some overlapping concepts here, e.g. I "think" that the Application.CR already contains the Namespace, isn't it?

Reply:
yes

Reply:
you might do something in another namespace for the app.